app/views/migration/_form.html.erb

```diff
@@ -1,5 +1,6 @@
 <% with_options scope: 'site.campaigns.migration.form' do |i18n| %>
-  <%= simple_form_for @aggregation, url: { action: 'create' }, builder: Migration::FormBuilder do |agg| %>
+  <% extra_form_html = { 'recaptcha-site-key': Recaptcha&.configuration&.site_key! } unless current_user %>
+  <%= simple_form_for @aggregation, html: extra_form_html, url: { action: 'create' }, builder: Migration::FormBuilder do |agg| %>
```

Comment:
In this branch, if JS is disabled in the browser, the form can not even be submitted, i.e. pressing enter key or submit button do nothing, whereas I would expect the form to be submitted then return validation errors, including one about the humanity validation requiring JS to be present.

Reply:
This is true, but since the recaptcha is entirely based on javascript(*there's a less secure no-script option), you won't ever be able to prove that you're not a robot unless you have javascript enabled. The acceptance criteria has been updated on the ticket to reflect this.